CHANGES.rst

```diff
@@ -4,9 +4,28 @@ FlexibleSUSY 2.6.2 [July, 08 2021]
 Changes
 -------
 
+* Parallelize calculation of decay amplitudes during the mathematica phase.
+  Example speedup:
+  .. list-table::
+   :widths: 25 25 50
+   :header-rows: 1
+
+   * - model
+     - sequential
+     - parallel
+   * - `SM`
+     -
```

Comment:
Will this table be filled before the merge?

Reply:
Yes. Sorry. I forgot about it. I'll copy a table from this PR.

Reply:
Thanks!